lib/core/imports/index.js

```diff
@@ -40,10 +40,36 @@ if (window.Uint32Array) {
   }
 }
 
+/**
+ * @deprecated
+ */
+class CSSSelectorParser {
+  constructor() {
+    this.parser = createParser({
+      syntax: {
+        pseudoClasses: {
+          definitions: {
+            Selector: ['is', 'not']
+          }
+        },
+        combinators: ['>'],
+        attributes: {
+          operators: ['^=', '$=', '*=', '~=']
+        }
+      }
+    });
+  }
+
+  parse(selector) {
+    return this.parser.parse(selector);
+  }
+}
+
 /**
  * Namespace `axe.imports` which holds required external dependencies
  *
  * @namespace imports
+ * @deprecated
  * @memberof axe
  */
-export { CssSelectorParser, doT, emojiRegexText, memoize, Color as Colorjs };
+export { CSSSelectorParser, doT, emojiRegexText, memoize, Color as Colorjs };
```

Comment:
Shouldn't update the casing of CSS otherwise this is a breaking change

```suggestion
export { CssSelectorParser, doT, emojiRegexText, memoize, Color as Colorjs };
```